tools/scripts/seed/user-data.js

```diff
@@ -12263,6 +12263,22 @@ module.exports.fullyCertifiedUser = {
     {
       completedDate: 1729240849345,
       id: '6763500bd5a85d5898cc21a9'
+    },
+    {
```

Comment:
added these 4 from the code editors (JS) module for the donation modal e2e tests